.gitignore

```diff
@@ -10,6 +10,8 @@ build/
 artifacts/
 src/**/obj/
 src/**/bin/
+buildscripts/**/obj/
+buildscripts/**/bin/
```

Comment:
Yes, it is using the new msbuild structure, so for anyone inclined to dive into the file system and add stuff here, it will be auto discovered by this project. 

Lovely for reducing friction when doing build work. Will eventually also double up as a candidate for housing custom msbuild tasks (let's hope that does not happen just yet).